cspell.json

```diff
@@ -0,0 +1,18 @@
+{
+    "language": "en",
+    "caseSensitive": false,
+    "dictionaries": [
+        "balena", 
+        "en_US", "en_GB",
```

Comment:
I like this, but I would actually support only one of them all over the docs to have consistency

Reply:
American English en_US